spec/std/exception_spec.cr

```diff
@@ -1,4 +1,16 @@
-require "spec"
+require "./spec_helper"
+
+private def compile_and_run_file(source_file)
+  with_tempfile("executable_file") do |executable_file|
+    Process.run("bin/crystal", ["build", "--release", "-o", executable_file, source_file])
```

Comment:
```suggestion
    Process.run("bin/crystal", {"build", "--release", "-o", executable_file, source_file})
```

Reply:
I don't think replacing array literals with tuples is always a good idea. Specially here because the args are still going to be converted to an array anyway in `Process.run`. And then the compiler also needs to create an entirely new type (the tuple) if it's not already there.

Reply:
Just, please don't request small changes with no semantic differences @Sija.